src/lib.rs

```diff
@@ -382,8 +382,8 @@ where
 
                     let mut args = Object::new();
                     if let (Some(file), Some(line)) = (callsite.file, callsite.line) {
-                        args.insert("[file]", file.to_string().into());
-                        args.insert("[line]", line.into());
+                        args.insert("file", file.to_string().into());
```

Comment:
The reason for the brackets was so it wouldn't conflict with any actual arguments (like those logged with `#[instrument]`).

I tried `.file` and `.line` and Perfetto and Chrome seem to be okay with those, so how about we go with that?

Reply:
Done!